src/Symfony/Component/TypeInfo/TypeFactoryTrait.php

```diff
@@ -54,6 +56,11 @@ public static function int(): BuiltinType
         return self::builtin(TypeIdentifier::INT);
     }
 
+    public static function intRange(int $from, int $to, bool $zeroIncluded = true): IntRangeType
```

Comment:
```suggestion
    public static function intRange(int $from = \PHP_INT_MIN, int $to = \PHP_INT_MAX): IntRangeType
```

Reply:
Ok, but keeing `$zeroIncluded`, right?

Reply:
Because of https://github.com/symfony/symfony/pull/59676#discussion_r1963108678, I think we can remove it

Reply:
Ah, good point. Sorry, I missed that one.